spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -93,7 +93,7 @@ spec:
               source /shared/p2p/p2p-addresses && \
               source /shared/config/service-addresses && \
               env && \
-              node --no-warnings /usr/src/yarn-project/aztec/dest/bin/index.js start --node --archiver --sequencer --pxe
```

Comment:
why drop the PXE?

Reply:
wasn't doing anything right? I believe it was only used for the `get-node-info` call from other services to check boo-node is up but this has now been added to Aztec Node API

Reply:
Hm yes, but I think this now conflicts with `spartan/aztec-network/files/config/config-validator-env.sh`, since it is asking for a PXE url. 

It is convenient in that case because when "dynamic boot node" is enabled, the validators use each other to bootstrap, so they use the PXE service, which is connected to the boot node if the network is public, and the validator service otherwise.

So I think the answer is to update the `wait-for-services` init container in the validators to use `echo "{{ include "aztec-network.validatorUrl" . }}" > /shared/boot_node/boot_node_url`, and then change back the `config-validator-env` to use the node url.

Could/should just be a separate issue though.